README.md

```diff
@@ -198,13 +198,9 @@ For a complete list of commands, see the
 ## Building the CLI
 ### Developing
```

Comment:
Maybe the overall leveling can be:
## Contributing to CLI
### Setting up workspace
### Building binary
### Adding new dependencies
### ....
### Running unit tests